.travis.yml

```diff
@@ -92,21 +93,59 @@ matrix:
+    - if [[ "$TRAVIS_OS_NAME" != "windows" ]] && [[ "$CI_MODE" == "test" ]]; then ./scripts/buildSuiteSparse.sh; fi
+    - if [[ "$TRAVIS_OS_NAME" != "windows" ]] && [[ "$CI_MODE" == "test" ]]; then ./scripts/buildSundials.sh; fi
+    - if [[ "$TRAVIS_OS_NAME" != "windows" ]] && [[ "$CI_MODE" == "test" ]]; then ./scripts/buildCpputest.sh; fi
+    - if [[ "$TRAVIS_OS_NAME" != "windows" ]] && [[ "$CI_MODE" == "test" ]]; then ./scripts/buildBNGL.sh; fi
+    - if [[ "$TRAVIS_OS_NAME" != "windows" ]] && [[ "$CI_MODE" == "test" ]]; then ./scripts/buildAmici.sh; fi
+    - if [[ "$TRAVIS_OS_NAME" != "windows" ]] && [[ "$CI_MODE" == "test" ]]; then ./scripts/installAmiciArchive.sh; fi
+    - if [[ "$TRAVIS_OS_NAME" != "windows" ]] && [[ "$CI_MODE" == "test" ]]; then ./scripts/installAmiciSource.sh; fi
+    # Enable Windows developer mode to support symlinks
+    #- if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then pip install --user -U pip; fi
+    # temporary test code
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command Get-ExecutionPolicy; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command Set-ExecutionPolicy Unrestricted -Force; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command Get-ExecutionPolicy; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command Write-Output $BASE_DIR; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command Write-Output "$BASE_DIR"; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command Get-ChildItem $BASE_DIR; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command Get-ChildItem .\\scripts; fi
+    # - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command Get-ChildItem ".\scripts"; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command Get-ChildItem ./scripts; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command Get-ChildItem $BASE_DIR\\scripts; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then powershell -Command New-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock" -Name AllowDevelopmentWithoutDevLicense -Value 1  -PropertyType DWord; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then git clone -c core.symlinks=true https://github.com/ICB-DCM/AMICI.git && cd AMICI; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then pip -v install python-libsbml; fi
+    - if [[ "$TRAVIS:OS:NAME" == "windows" ]]; then powershell .\\scripts\\installOpenBLAS.ps1; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]] && [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then git checkout -qf $TRAVIS_COMMIT; fi
+    - if [[ "$TRAVIS_OS_NAME" == "windows" ]] && [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then git fetch origin pull/$TRAVIS_PULL_REQUEST/head:$TRAVIS_BRANCH && git checkout $TRAVIS_BRANCH; fi
```

Comment:
https://github.com/ICB-DCM/AMICI/pull/847#issuecomment-558057593 @thomassligon : below this line